TenureInformationApi/V1/Factories/ResponseFactory.cs

```diff
@@ -5,9 +5,9 @@
 
 namespace TenureInformationApi.V1.Factories
 {
-    public static class ResponseFactory
+    public class ResponseFactory : IResponseFactory
```

Comment:
Was there a reason you changed this class from a static one to one injected by the DI container?

Reply:
I've been able to get it working without injecting the DI container